### ISSUE TITLE
[Bug] Coordinates Bug for Addresses in TIA_request

### ISSUE THREAD

**Issue by ChrispinP** [cityofaustin/atd-data-tech#6663]
- [x] Create New Address Fields
- [x] Update Forms
- [x] Update Rules
- [x] Test Coordinate data

**Comment by ChrispinP** [comment]
New Address fields created. Forms and rules updated. Tested coordinate data and updated formulas. Bug is now resolved. Removing old address fields.

![image](https://user-images.githubusercontent.com/67915167/125511075-cc355f22-6257-48dd-9df2-bc2b9d962d29.png)

Ready to review

**Comment by ChrispinP** [comment]
Moving back to in progress as Knack continues to investigate bug. They concur they dont think its the fields, and they are conferring with the map provider as well.

I may spend some more time testing to see if I can better pinpoint the bug.

**Comment by amenity** [comment]
Waiting for Knack support to help with this.

**Comment by ChrispinP** [comment]
sent Knack another message on a status update for this issue. Still no response in 3 weeks. May need to submit a new issue again.

**Comment by ChrispinP** [comment]
submitted a new issue to Knack since I still received no response.

**Comment by ChrispinP** [comment]
I have finally figured out this bug while updating the TDS Portal.

When you set an Address field to a Text Formula field and a Short Text field, one overrides the other on Record update and corrupts the record. Therefore, to capture coordinates, we only set an Address field to a Short Text field.
![image](https://user-images.githubusercontent.com/67915167/147299662-d88e6f64-b40f-4755-866e-44488eddee57.png)

I will reach out to Knack to see if we can "un-corrupt" these TIA records. This fix will work for all new records created.
